toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -7,18 +7,27 @@
 # $3 path to find RPMs. May be in PATH/<arch>/*.rpm
 # $4 path to log directory
 
-[ -n "$1" ] && [ -n "$2" ] && [ -n "$3" ] && [ -n "$4" ] || { echo "Usage: create_worker.sh <./worker_base_folder> <rpms_to_install.txt> <./path_to_rpms> <./log_dir>"; exit; }
+[ -n "$1" ] && [ -n "$2" ] && [ -n "$3" ] && [ -n "$4" ] && [ -n "$5" ] && [ -n "$6" ] || { echo "Usage: create_worker.sh <./worker_base_folder> <rpms_to_install.txt> <./path_to_rpms> <./log_dir> <./bldtracker>"; exit; }
 
 chroot_base=$1
 packages=$2
 rpm_path=$3
 log_path=$4
+bldtracker=$5
+timestamp_dir=$6
 
 chroot_name="worker_chroot"
 chroot_builder_folder=$chroot_base/$chroot_name
 chroot_archive=$chroot_base/$chroot_name.tar.gz
 chroot_log="$log_path"/$chroot_name.log
 
+$bldtracker \
+    --script-name="create_worker_chroot.sh" \
+    --step-name="Start running the script" \
+    --dir-path=$timestamp_dir \
```

Comment:
Nit-pick: best to surround paths with quotes (same what you did a line below) in case someone decides to use some unusual characters like a space.
```suggestion
    --dir-path="$timestamp_dir" \
```

Reply:
In general please either update the code or mention why you're resolving a comment. Otherwise, it's hard to understand what happened (maybe you disagree, we could have discussed this offline and this is no longer needed, etc.).

Reply:
Ping.